drone-fly-app/src/main/java/com/expediagroup/dataplatform/dronefly/app/context/CommonBeans.java

```diff
@@ -65,8 +76,21 @@ public ListenerCatalog listenerCatalog(HiveConf conf) throws MetaException {
 
   @Bean
   public MessageReaderAdapter messageReaderAdapter() {
-    KafkaMessageReader delegate = KafkaMessageReaderBuilder.builder(bootstrapServers, topicName, instanceName).build();
+    Properties clientProperties = getConsumerProperties();
+    KafkaMessageReader delegate = KafkaMessageReaderBuilder.
+            builder(bootstrapServers, topicName, instanceName).
+            withConsumerProperties(clientProperties).
+            build();
     return new MessageReaderAdapter(delegate);
   }
 
-}
+  private Properties getConsumerProperties() {
+    Properties consumerProperties = new Properties();
+    getEnvProperties().forEach((key, value) -> {
+        consumerProperties.put(key.toString(), value.toString());
```

Comment:
You still need to strip of the prefix here I would think

Reply:
When doing @ConfigurationProperties(CONSUMER_PROPERTIES_PREFIX) the properties map actually removes the prefix from the keys so no need to validate

Reply:
ok cool